metricbeat/module/prometheus/_meta/prometheus.yml

```diff
@@ -0,0 +1,32 @@
+# my global config
+global:
+  scrape_interval:     15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# Alertmanager configuration
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      # - alertmanager:9093
+
+# Load rules once and periodically evaluate them according to the global 'evaluation_interval'.
+rule_files:
+  # - "first_rules.yml"
+  # - "second_rules.yml"
+
+remote_write:
+  - url: "http://REMOTE/write"
```

Comment:
Is it a problem if nothing is listening on this remote?

Reply:
I saw that Prometheus will just keep reporting that was not able to send data the `remote_write`.